static/icons/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "TBA",
```

Comment:
Sure...I guess that location makes more sense logically. Let me pull in the color and icon information.

On the existing version, it shows `name `and `short_name` both as `The Blue Alliance` is that what you want? Or should short name be `TBA`?